list/List.js

```diff
@@ -318,22 +364,34 @@ define([
 				|| (this._isCategorized()
 						&& ("categoryAttr" in props || "categoryFunc" in props || "categoryRenderer" in props))) {
 				if (this._dataLoaded) {
-					this._setBusy(true, true);
+					this._busy = true;
 
 					// trigger a reload of the list
 					this.notifyCurrentValue("source");
 				}
 			}
+			if (("renderItems" in props && this.renderItems) || "_busy" in props || "showNoItems" in props) {
+				this._updateListView();
+			}
+		},
+
+		postRender: function () {
+			// moving down to the containerNode any aria attribute that has been set to the root node.
+			for (var i = 0; i < this.attributes.length; i++) {
+				if (/^aria-/.test(this.attributes[i].name)) {
+					this.containerNode.setAttribute(this.attributes[i].name, this.attributes[i].value);
+					this.removeAttribute(this.attributes[i].name);
```

Comment:
If you update to override `removeAttribute()` (as I mentioned above) then this line will need to be changed to

```
// force remove from root node not container node
HTMLElement.prototype.removeAttribute.call(this, this.attributes[i].name);
```